### DISCUSSION TITLE
blog/how-we-do-loadtest-shopware/

### DISCUSSION THREAD

**Discussion by giscus** [shyim/shyim.me#50] [Q&A]
# blog/how-we-do-loadtest-shopware/

With the Shopware version 6.4.11.0, we improved the performance overall a lot. To make sure that we still improve, we decided to set up a nightly load testing job to ensure that the application performance stays the same. So we have running since a month an automated load test for Shopware 6 running, and in this post, I want to speak about it. To test the Shopware, we use multiple dedicated Cloud Servers from Hetzner Cloud and wrote some custom tooling around Ansible to spawn the servers before.

https://shyim.me/blog/how-we-do-loadtest-shopware/

**Comment by michael-sfx** [comment]
Hey interesting article. However I could not find any info about those 2 settings:

'FEATURE_NEXT_13250' => 1,
 // Enable breaking performance optimizations
 'PERFORMANCE_TWEAKS' => 1,

What are they used for?

Cheers
Mike
